media/js/base/core-datalayer.js

```diff
@@ -2,50 +2,34 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-// create namespace
-if (typeof Mozilla == 'undefined') {
```

Comment:
Because this is relying & adding to an existing namespace that's now in a different bundle altogether, I think we should probably keep this check, and also add a fallback for if `Mozilla.Analytics` is undefined also?